locales/en-US.json

```diff
@@ -280,5 +280,6 @@
   "writing": "Writing",
   "year": "Year",
   "years": "Years",
-  "youMayAlsoLike": "You may also like"
+  "youMayAlsoLike": "You may also like",
+  "stopPlayback": "Stop playback"
```

Comment:
Can we keep this file in alphabetical order, please.

`i18n-ally` for VSCode should do this automatically for you when adding translations through it.